ansible/roles/jupyterhub/files/jupyterhub_config_lti11.py

```diff
@@ -43,34 +43,12 @@
 
 c.JupyterHub.cookie_secret_file = os.path.join(data_dir, 'jupyterhub_cookie_secret')
 
-# The instructor1 and instructor2 users have access to different shared
-# grader notebooks. bitdiddle, hacker, and reasoner students are from
-# the `nbgrader quickstart <course name>` command.
-c.JupyterHub.load_groups = {
-    os.environ.get('DEMO_INSTRUCTOR_GROUP'): [
-        'instructor1',
-        'instructor2',
-        os.environ.get('DEMO_GRADER_NAME'),
-    ],  # noqa: E231
-    os.environ.get('DEMO_STUDENT_GROUP'): ['student1', 'bitdiddle', 'hacker', 'reasoner',],  # noqa: E231
-}
-
 # Allow admin access to end-user notebooks
 c.JupyterHub.admin_access = True
 
-# Start the grader notebook as a service. The port can be whatever you want
-# and the group has to match the name of the DEMO_GRADER_NAME group defined above.
-# The cull_idle service conserves resources.
-course_id = os.environ.get('COURSE_ID')
+# Define some static services that jupyterhub will manage
 announcement_port = os.environ.get("ANNOUNCEMENT_SERVICE_PORT") or '8889'
```

Comment:
Let's use single quotes by default to provide more consistency.

Reply:
I agree